### PR TITLE
Also print err

### PR DESCRIPTION
**Pull request by filipesilva**

### CODE REVIEW

**Comment by borkdude** [comment]
Wouldn't it be better to print `err` to `*err*` instead of `*out*`?

**Comment by filipesilva** [comment]
You know that was my first try. I bound *err* to *out* and called print. But I didn't see any output and didn't debug further, so I just did the simple thing instead.

I don't have a strong opinion either way though. I'm just using this repl client on a script that spawns a clojure process then interacts with it via repl, for quick startup, and wanted to see errors.

**Comment by borkdude** [comment]
> I bound err to out and called print. 

Try the other way around. `println` prints to `*out*` so you need to bind that to `*err*`, not the other way around

**Comment by filipesilva** [comment]
Sorry that's what I meant I did, am just on the phone. I'll try again
though.

**Comment by filipesilva** [comment]
Well that might have been what I meant but it looks like it wasn't what I tested before. Tested again and it worked fine. PR updated!

File: src/impl/babashka/nrepl_client.clj [hunk 1]
```diff
@@ -49,8 +49,12 @@
...
       (let [{:keys [status out err value]} (read-reply in session id)]
-        (when (or out err)
-          (print (or out err))
+        (when out
+          (print out)
+          (flush))
+        (when err
+          (binding [*out* *err*]
+            (print err))
           (flush))
```
Comment (borkdude) [inline src/impl/babashka/nrepl_client.clj]: I believe this flush should also be inside the binding form

Comment (filipesilva) [reply src/impl/babashka/nrepl_client.clj]: Updated

**Comment by borkdude** [comment]
Thanks!

**Comment by filipesilva** [comment]
Thank you for making it! This is how I'm using it in case you're wondering: https://github.com/filipesilva/fdb/blob/aea0818e3a238b11dfb38b531523e7c4628d1470/src/fdb/bb/cli.clj#L47-L50

**Comment by borkdude** [comment]
Nice, what is that project about?

**Comment by filipesilva** [comment]
It makes a xtdb from files and metadata on disk, watches them for changes, and supports triggered behaviour (e.g. call this fn when the file changes, of when this xtdb query changes). Haven't gotten around to making a good readme yet. Mostly use it for automation, and to trigger stuff on my laptop by editing files on my phone.

**Comment by filipesilva** [comment]
I really wanted a fast CLI start so went with a bb script that I can symlink, then it starts a background clojure process and talks to it via nrepl. Some functionality also needs to connect to the existing running process so it was a nice fit.

**Comment by borkdude** [comment]
cool :)

**Pull request merged**